reo/tests/posts/test_curtailment_POST.json

```diff
@@ -0,0 +1,30 @@
+{"Scenario": { 
```

Comment:
@zolanaj one more comment here - Instead of the simplified post can you provide a POST that has all the defaults filled in? (i.e. what you get back from the inputs section of the results endpoint. Note you may need to strip out percent_share and the prod_factor inputs because these can cause validation errors unless you've already pulled the latest from develop here). This way when defaults change later they won't break the test suite.

Reply:
@t-kwasnik  I found a bug in one of the constraints, and I believe we are good now.  I get the following results from your post:

api_response['outputs']['Scenario']['Site']['Wind']['year_one_to_load_series_kw'][2441:2541]
​
[6.685, 6.387, 13.554, 12.901, 11.045, 10.865, 10.326, 9.925, 5.753, 9.569, 11.824, 12.033, 0.0, 3.795, 0.0, 0.0, 0.0, 20.691, 20.154, 20.786, 2.043, 0.0, 19.587, 0.0, 9.662, 4.716, 13.451, 12.782, 10.963, 7.024, 4.943, 3.452, 10.626, 10.502, 12.464, 12.365, 0.0, 8.791, 0.0, 20.433, 18.401, 13.804, 10.157, 6.507, 0.844, 0.0, 0.0, 0.0, 6.699, 15.499, 13.097, 6.957, 4.421, 0.0, 9.661, 9.608, 9.565, 9.671, 0.0, 0.0, 0.0, 3.893, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 6.2, 8.473, 11.307, 11.325, 5.629, 3.246, 1.731, 1.715, 1.653, 0.0, 0.0, 0.0, 0.0, 0.0, 0.727,0.832, 0.0, 3.053, 0.0, 0.0, 13.206, 0.0, 0.0, 0.0, 0.0, 10.848, 0.0, 10.802]

which is in line with the loads you present above.  

Could you also let me know if the revised post is sufficient?  I didn't include any time-series inputs, for example, though I didn't really find defaults for those.